ui/guest/src/classes/ContentController.ts

```diff
@@ -958,19 +961,21 @@ function reducer(lookupTable: LookupTable<ContentInstance>, model: ContentInstan
 
 }
 
-function getResult(model: ContentInstance, fieldId: string, index: string | number): number[] | string[] {
-  const isStringIndex = typeof index === 'string';
+//function getResult(model: ContentInstance, fieldId: string, index: string | number): number[] | string[] {
+function getResult(collection: string[], index): string[] {
```

Comment:
This looks temporary. What's the idea here?

Reply:
sometimes we need to collection(insert item), but others we need the result( collection - the sort item)